packages/preview-middleware-client/src/cpe/init.ts

```diff
@@ -9,9 +9,16 @@ import { SelectionService } from './selection';
 import { ChangeService } from './changes/service';
 import { loadDefaultLibraries } from './documentation';
 import Log from 'sap/base/Log';
+import { enableTelemetry } from '@sap-ux-private/control-property-editor-common';
 
 export default function init(rta: RuntimeAuthoring): Promise<void> {
     Log.info('Initializing Control Property Editor');
+    
+    // enable telemetry if requested
+    const flexSettings = rta.getFlexSettings();
+    if (flexSettings.telemetry === true) {
```

Comment:
I'm not sure if we should add our own properties to  `flexSettings`, since it's later passed to RTA when creating changes.

Reply:
It's not affecting anything else. I checked.

Reply:
@voicis if it is ok with you, please mark it as resolved.